site/pages/institution-landing-en.hbs

```diff
@@ -0,0 +1,188 @@
+</div>
+
+<div class="well well-sm brdr-0 mrgn-bttm-0">
+	<div class="container">
+		<section class="provisional gc-about">
+			<h2 class="h3 mrgn-tp-md">Contact us</h2>
+			<ul class="wb-eqht list-unstyled mrgn-tp-lg lst-spcd-2 colcount-md-3">
+				<li><a href="#">Contact [Institution]</a></li>
+				<li><a href="#">[Top contact task 2]</a></li>
+				<li><a href="#">[Top contact task 3]</a></li>
+			</ul>
+		</section>
+		<div class="row">
+			<section class="col-md-8 pull-left mrgn-bttm-lg provisional gc-about">
+				<h2 class="h3 mrgn-tp-md">About the [Institution]</h2>
+				<ul class="list-bld list-unstyled mrgn-tp-lg lst-spcd-2 colcount-md-2">
+					<li><a href="#">Mandate</a></li>
+					<li><a href="#">Organizational structure</a></li>
+					<li><a href="#">Transparency</a></li>
+					<li><a href="#">Job opportunities</a></li>
+					<li><a href="#">Reports</a></li>
+					<li><a href="#">Compliance</a></li>
+					<li><a href="#">Enforcement notifications</a></li>
+					<li><a href="#">[Institutional link]</a></li>
+					<li><a href="#">[Institutional link]</a></li>
+					<li>More: <a href="#">About the [Institution]</a></li>
+				</ul>
+			</section>
+			<div class="provisional col-xs-12 col-md-4 col-sm-5 pull-right">
+				<section class="lnkbx">
+					<h2 class="h3">Minister</h2>
+					<p>
+						<a href="#">[(Honourable) first and last name]</a><br>
+						<small>Minister's official title</small>
+					</p>
+					<p>
+						<a href="#">[(Honourable) first and last name]</a><br>
+						<small>Minister's official title</small>
+					</p>
+					<p>
+						<a href="#">[(Honourable) first and last name]</a><br>
+						<small>Minister's official title</small>
+					</p>
+				</section>
+				{{>follow-us}}
+			</div>
+		</div>
+	</div>
+</div>
+
+<div class="container">
+	<div class="row">
+		<section class="provisional wb-feeds gc-feeds limit-3 col-md-4">
+			<h2 class="h3">News</h2>
+			<ul class="feeds-cont list-unstyled lst-spcd">
+				<li>
+					<a data-ajax="demos/feeds/demo/example-en.atom.json" href="#">
+					[Institution] news
+					</a>
+				</li>
+			</ul>
+			<p>More: <a href="#">[Institution] news</a></p>
+		</section>
+		<section class="col-md-8 gc-features">
+			<h2 class="h3">Features</h2>
+			<div class="row wb-eqht">
+				<div class="col-sm-6">
+					<div class="well well-sm brdr-rds-0 eqht-trgt">
+						<img class="img-responsive full-width" src="{{assets}}/img/360x203.png" alt="" />
+						<h3 class="h5"><a class="stretched-link" href="#">[Feature hyperlink text]</a></h3>
+						<p>Brief description of the feature being promoted.</p>
+					</div>
+				</div>
+				<div class="col-sm-6">
+					<div class="well well-sm brdr-rds-0 eqht-trgt">
+						<img class="img-responsive full-width" src="{{assets}}/img/360x203.png" alt="" />
+						<h3 class="h5"><a class="stretched-link" href="#">[Feature hyperlink text]</a></h3>
+						<p>Brief description of the feature being promoted.</p>
+					</div>
+				</div>
+			</div>
+		</section>
```

Comment:
@delisma In the beta template your team provided, the features links don't have underline and the text underneath is small. I was wondering if this would be something you guys want to implement as the standard for features of type "Government activities" or if it is just a one-off. If it is just a one-off, than the current GCWeb look prevails. If not, then we will still treat this as a separate request but I wanted to throw it out there.

Reply:
@pcwsmith, @jmealing there is a request with PP to normalize the different gc-features card patterns (with wells and without wells). Do we want a specific implementation?

Reply:
@delisma 

The feature links should have an underline, the last I reviewed the page it was there. Yes, the text should be smaller so it is the same text size as doormats. We had asked for this to be corrected with the "Government initiatives" on the home page, but the issue wasn't pressed since it wasn't urgent. These designs should match IMO (other than it's occupying different space in the columns to make room for the news). It was my understanding this would become the "standard" with the well

There is the provisional single feature promotion that is meant to look integrated with no well, but that's the intention, to look contextual and part of the page, where the others are clearly promotional